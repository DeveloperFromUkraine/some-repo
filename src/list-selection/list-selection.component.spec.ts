import { ComponentFixture, TestBed } from "@angular/core/testing";
import { ListSelectionComponent } from "index";
import { ComponentTest } from "../../test/test-bed/component";

describe('ListSelectionComponent', () => {
    let fixture: ComponentFixture<ListSelectionComponent>;

    beforeEach(async () => {
        await ComponentTest.createTestBed([], [ListSelectionComponent]);
    });

    beforeEach(() => {
        fixture = TestBed.createComponent(ListSelectionComponent);
    });

    it('should match snapshot', () => {
        expect(fixture).toMatchSnapshot();
    });
});