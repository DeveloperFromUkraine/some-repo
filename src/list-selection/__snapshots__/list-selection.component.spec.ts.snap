// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List Selection should match snapshot 1`] = `
<ign-list-selection>
  <div
    class="container"
  />
</ign-list-selection>
`;
